index.bs

```diff
@@ -12264,8 +12264,8 @@ in which case they are exposed on every object that [=implements=] the interface
         <emu-val>undefined</emu-val>; there is no [=attribute setter=] function.
     1.  Assert: |attribute|'s type is not a [=promise type=].
     1.  Let |steps| be the following series of steps:
-        1.  If no arguments were passed, then [=JavaScript/throw=] a <l spec=ecmascript>{{TypeError}}</l>.
-        1.  Let |V| be the value of the first argument passed.
+        1.  If no arguments were passed, then let |V| be <emu-val>undefined</emu-val>.
+        1.  Otherwise, let |V| be the value of the first argument passed.
```

Comment:
Nit: it's not great to scope a "let" inside if/otherwise blocks. Prefer this style:

> 1. Let V be undefined.
> 2. If any arguments were passed, then set V to the value of the first argument passed.